### PR TITLE
Add unit test for pkg/chart/chart.go

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@cn.fujitsu.com>

### CODE REVIEW

**Review by jdolitsky — APPROVED** [review]

File: pkg/chart/chart_test.go [hunk 1]
```diff
@@ -78,3 +78,13 @@ func TestSaveChartNoRawData(t *testing.T) {
 
 	is.Equal([]*File(nil), res.Raw)
 }
+
+func TestName(t *testing.T) {
```
Comment (hickeyma) [inline pkg/chart/chart_test.go]: Should this be renamed TestMetadata? Then the test would check the rest of the properties of metadata.
I am not sure of the value of just checking the name only.

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for push the test @hs0210. Got some suggestions on improving the test to include other properties of the metadata.

**Comment by hs0210** [comment]
PTAL

File: pkg/chart/chart_test.go [hunk 2]
```diff
@@ -78,3 +78,21 @@ func TestSaveChartNoRawData(t *testing.T) {
 
 	is.Equal([]*File(nil), res.Raw)
 }
+
+func TestMetadata(t *testing.T) {
+	chrt := Chart{
+		Metadata: &Metadata{
+			Name:       "foo.yaml",
+			AppVersion: "1.0.0",
+			APIVersion: "1.0.0",
```
Comment (hickeyma) [inline pkg/chart/chart_test.go]: Can you set the `APIVersion` to v2?

**Review by hickeyma — CHANGES_REQUESTED** [review]
Thanks for update @hs0210. Small nit to fix.

**Comment by hs0210** [comment]
PTAL

File: pkg/chart/chart_test.go [hunk 3]
```diff
@@ -78,3 +78,21 @@ func TestSaveChartNoRawData(t *testing.T) {
 
 	is.Equal([]*File(nil), res.Raw)
 }
+
+func TestMetadata(t *testing.T) {
+	chrt := Chart{
+		Metadata: &Metadata{
+			Name:       "foo.yaml",
+			AppVersion: "1.0.0",
+			APIVersion: "2.0.0",
```
Comment (hickeyma) [inline pkg/chart/chart_test.go:87]: Do you mind setting it as follows: `APIVersion: "v2"`? It is to signify the chart v2 format

**Review by hickeyma — CHANGES_REQUESTED** [review]
Small nit still on the `APIVersion`.

**Comment by hs0210** [comment]
I've changed it. PTAL, Thanks.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @hs0210

**Pull request merged**